R/Functions_L0_auxilary.R

```diff
@@ -2536,46 +2516,6 @@ getDronesCsdGeno <- function(x, nInd = NULL, dronesHaploid = TRUE,
 #'
 #' @return logical
 #'
-#' @examples
```

Comment:
@gregorgorjanc , this was required by CRAN - either export the functions or remove the examples. It didn't help that I had them in dontrun

Reply:
I don’t like removing examples as they serve as a test/debug case for future. You I know that it takes half an hour to whizz these up properly. Isn’t there a way not to remove them? Comment them out maybe?

Reply:
No, code in the comments should not be commented out - that was another comment from the CRAN. What we could do is rename the PR where I removed them - so that we can easily find them when we want to get them back

Reply:
FFS! Can you please stick the example within the function commented out? Ugly, but at least we don’t loose the example.

Reply:
What if I create another file within R, put examples in it and put it in .Rbuildignore?